ambry-server/src/integration-test/java/com.github.ambry.server/RouterServerTestFramework.java

```diff
@@ -358,7 +358,12 @@ void check() throws Exception {
    * @param opChain the {@link OperationChain} object that this operation is a part of.
    */
   private void startGetBlobAuthorizationFailTest(final OperationChain opChain) {
-    Callback<GetBlobResult> callback = new TestCallback<>(opChain, false);
+    Callback<GetBlobResult> callback = new TestCallback<GetBlobResult>(opChain, false) {
```

Comment:
Are you expecting an error to occur here? If so, you could set `expectError` to true, which will let the chain continue

Reply:
e.g. `Callback<GetBlobResult> callback = new TestCallback<GetBlobResult>(opChain, true);`

Reply:
oh. sure.